fern/openapi-ir/definition/ir.yml

```diff
@@ -0,0 +1,136 @@
+types: 
+  IntermediateRepresentation: 
+    properties: 
+      displayName: optional<string>
+      environments: list<Environment>
+      endpoints: list<Endpoint>
+      schemas: map<SchemaId, Schema>
+      schemasToTags:
+        type: map<SchemaId, list<TagId>>
+        docs: Which schemas are referenced by which tags
+
+  WithDescription: 
+    properties: 
+      description: optional<string>
+
+  Environment: 
+    extends: WithDescription
+    properties: 
+      name: optional<string>
+      url: string
+
+  TagId: 
+    type: string
+    docs: The name of the tag.
```

Comment:
`TagName` or `TagId`?

Reply:
it's a unique name so wasn't sure -- decided to call them ids (see `SchemaId` but i can switch)